### PR TITLE
Feature/586 scenarioo maven central

### PR DESCRIPTION
**Pull request by danielsuter**
Added deployment to sonatype oss to enable deployment to maven central.
Renamed scenarioo-server artifact to scenarioo-viewer. Don't use github version anymore, since we want to do semantic versioning anyways.
Reference scenarioo-java from maven central and sonatype oss for snapshot builds.
Fixed some svg.js dependencies as their repository has been moved.

e2e tests are green on my machine locally.

### CODE REVIEW

**Comment by bruderol** [comment]
Please consider my last comment here: https://github.com/scenarioo/scenarioo/issues/586

@adiherzog Can we check that our CI build will still work with the changed artifact name? Is there a way that we could start to run our build system for pull requests? That would be really helpful for things like that.

File: build.gradle [hunk 1]
```diff
@@ -20,6 +20,8 @@ ext.scenariooAggregatedDataFormatVersion = '2.1.0'
  */
 ext.scenariooReleaseBranch = 'release-3.0'
 
+ext.scenariooVersion = '3.0.1-SNAPSHOT'
```
Comment (danielsuter) [inline build.gradle]: I don't see a good reason to use the git commit id. We agreed on using semantic version. Our artifacts should be named accordingly. It only makes our life harder (e.g. we always have to produce a scenarioo-latest to be able to deploy our CI).

File: scenarioo-client/bower.json [hunk 2]
```diff
@@ -12,10 +12,10 @@
     "angular-bootstrap": "~1.2.2",
     "font-awesome": "~3.2.1",
     "twigs": "0.1.3.1",
-    "svg.js": "~2.0.4",
-    "svg.draggable.js": "~2.0.0",
-    "svg.select.js": "https://github.com/parabolika66/svg.select.js.git#~1.0.8",
-    "svg.resize.js": "https://github.com/parabolika66/svg.resize.js.git#~1.1.3",
+    "svg.js": "https://github.com/svgdotjs/svg.js.git#2.0.4",
+    "svg.draggable.js": "https://github.com/svgdotjs/svg.draggable.js.git#2.0.0",
+    "svg.select.js": "https://github.com/Fuzzyma/svg.select.js.git#1.0.5",
```
Comment (danielsuter) [inline scenarioo-client/bower.json]: Because it's dead.

File: build.gradle [hunk 3]
```diff
@@ -1,7 +1,7 @@
 /**
  * Used Version of Scenarioo Java Library for reading scenarioo docu data
  */
-ext.scenariooApiVersion = '2.1.0'
+ext.scenariooApiVersion = '2.1.1-SNAPSHOT'
```
Comment (danielsuter) [inline build.gradle]: Ok, I've switched back to the released version.

**Comment by bruderol** [comment]
@danielsuter and me discussed it personally - what is the state dani? do we have to take any action or can we merge that soon?

**Comment by bruderol** [comment]
**CAUTION: Please do currently not merge this branch until the problems due to changed dependencies are fixed.** We want to make another minor release of the viewer soon with some improvements of the diff viewer see other open pull request and issue #591 ! Thanks for taking care of this @danielsuter

**Comment by danielsuter** [comment]
@bruderol The state is a a dependency, which we can only fix by cloning the dependency and reapply the fixes. This sucks.

**Comment by bruderol** [comment]
yes, I know.

**Comment by bruderol** [comment]
Thanks a lot @danielsuter ! 👍

**Pull request merged**